assets/app/views/_overview-deployment.html

```diff
@@ -76,13 +76,18 @@ <h3 class="h4">{{rc.metadata.name}}</h3>
     <div class="component-label">Pod Template</div>
 
     <pod-template
-      pod-template="rc.spec.template" 
+      pod-template="rc.spec.template"
       images-by-docker-reference="imagesByDockerReference"
       builds="builds">
     </pod-template>
 
     <!-- TODO: surface spec.replicas/status.replicas here? -->
-    <div class="component-label">Pods</div>
+    <div class="component-label">
+      Pods
+      <span ng-if="rc.status.replicas === rc.spec.replicas" class="component-status">({{rc.status.replicas}})</span>
+      <span ng-if="rc.status.replicas !== rc.spec.replicas" class="component-status">({{rc.status.replicas}} current, {{rc.spec.replicas}} desired)</span>
```

Comment:
Check behavior when you scale up a lot, then set spec.replicas to 0. If I'm reading the API correctly, when spec.replicas is "0" it gets omitted from the API, when status.replicas is "0", it gets included. Don't want to show `X current, desired`